docs/ReleaseFlow.md

```diff
@@ -0,0 +1,31 @@
+# Release Flow
+
+## Requirements
+| Name | Version | Mandatory | link |
+|:------|:-------|:----------|:------|
+| Helm | 3.2.1 or latest | no | https://helm.sh/docs/intro/install/ |
+
+## Preparation
+You need to set the `version` of the `Chart.yaml` of each `helm-charts` to the version to be released.
+``` yaml
+version: 2.1.0
+```
+
+## Release
+Run the release manually from the Github UI.
+* [Release Action](https://github.com/scalar-labs/helm-charts/actions/workflows/release.yml)
+
+## Points of Attention
+### If the version of the `main` branch of the `charts` you are releasing is higher than the version you are releasing
```

Comment:
I think it's simpler to always follow the process. 
What do you think?
It would be great if we can chat about it tomorrow.